libs/langchain-scripts/src/build_v2.ts

```diff
@@ -1,10 +1,10 @@
 import { spawn } from "node:child_process";
 import ts from "typescript";
 import fs from "node:fs";
-import { rimraf } from "rimraf";
 import { Command } from "commander";
-import { rollup } from "rollup";
+import { rollup } from "@rollup/wasm-node";
```

Comment:
`rollup` does not work the same on windows, see [here](https://github.com/rollup/rollup/issues/5571#issuecomment-2220490077)